test/common/src/main/kotlin/org/codice/compliance/verification/binding/RedirectBindingVerifier.kt

```diff
@@ -55,10 +96,10 @@ class RedirectVerifier(val response: IdpRedirectResponse) : BindingVerifier() {
     /**
      * Verifies the redirect response has no XMLSig in the url according to the redirect binding rules in the binding
      * spec
-     * 3.4.4.1 DEFLATE ENCODING
+     * 3.4.4.1 DEFLATE Encoding
      */
     private fun verifyNoXMLSig() {
-        if (response.responseDom.children("Signature").isNotEmpty()) {
+        if (response.responseDom!!.children("Signature").isNotEmpty()) {
```

Comment:
❗️ This can potentially through an NPE (protection against which is, please recall, one of the primary advantages that Kotlin provides over Java).

See note on `IdpResponseDecorators.kt`.